esp/templates/program/modules/commmodule/step2.html

```diff
@@ -38,7 +38,8 @@
 Please write your email/message now. Note that you can use <tt>{{ parameter }}</tt> to enter a parameter
 into the text of the message&mdash;the parameters are listed below.
 
-<p><b>Tip:</b> If you would like to send your message in HTML format, wrap your entire message body in &lt;html&gt; and &lt;/html&gt; tags.</p>
+<p><b>Tip:</b> The default {{ settings.ORGANIZATION_SHORT_NAME }} template (which you can modify <a href="/admin/utils/templateoverride/">here</a>) is automatically wrapped around your message below.
+Don't forget to write the body in HTML, but no need for &lt;html&gt; and &lt;/html&gt; tags.</p>
```

Comment:
We should still allow writing plain-text emails, because it's easier and if you don't need any formatting it's better.  (Ideally we may want to switch to Markdown for emails, because that's both easier and can be used to make pretty things.)